tests/admin_checks/tests.py

```diff
@@ -37,6 +39,14 @@ def check(self, **kwargs):
         return ['error!']
 
 
+class MyAuthMiddleware(AuthenticationMiddleware):
```

Comment:
Doesn't matter much, but something like `AuthenticationMiddlewareSubclass` would be more descriptive about the purpose.